### ISSUE TITLE
False positive: The condition of `if` should not be wrapped in parentheses 

### ISSUE THREAD

**Issue by jueberschlag** [rrrene/credo#461]
### Environment

* Credo version (`mix credo -v`): 0.8.6
* Erlang/Elixir version (`elixir -v`): 1.5.1
* Operating system: Ubuntu

### What were you trying to do?
I tried to use Bitwise operators in if condition

```elixir
defmodule Example do
	@moduledoc false
	import Bitwise

	def bar(foo, bar) do
		if (foo &&& 0b1000) > 0, do: bar, else: nil
	end

	def foobar(foo) do
		foo
	end
end
```

### Expected outcome
No `Code Reliability` warning

### Actual outcome
```
The condition of `if` should not be wrapped in parentheses
```
Something strange also: the warning disappear if the function containing the if-Bitwise operation is the last functions of the module.

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍
